Source/Extensions/Blazorise.QRCode/QRCodeJSOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Blazorise.QRCode;
+
+public class QRCodeInitializeAndUpdateJSOptions
```

Comment:
Just name it `QRCodeJSOptions`. If there is a need for an update option, then we will introduce them in the future as a new type.